ratis-test/src/test/java/org/apache/ratis/datastream/TestDataStreamNetty.java

```diff
@@ -96,6 +96,7 @@ private void testCloseStream(int leaderIndex, int numServers) throws Exception {
       RaftPeerId peerId = RaftPeerId.valueOf("s" + i);
       RaftProperties properties = new RaftProperties();
       NettyConfigKeys.DataStream.setPort(properties, NetUtils.createLocalServerAddress().getPort());
+      RaftConfigKeys.DataStream.setType(properties, SupportedDataStreamType.NETTY);
```

Comment:
This line is redundant. If you check line 61 this config is already be set.

Reply:
hmmm I see `RaftProperties properties = new RaftProperties();` is not using the existing `properties`. 

I will take a look and maybe fix this in my branch.

Reply:
@amaliujia It's a mock RaftServer, will not use properties set at  line 61.

Reply:
O I see. thanks for clarification